web/package.json

```diff
@@ -31,23 +31,23 @@
   "dependencies": {
```

Comment:
❓ Did you only upgrade native dependencies (and those you had to in other workspaces) or basically all dependencies? Just asking whether we need separate issues to upgrade the other dependencies.

Reply:
Yes, mainly only the native ones, I'll open an issue for web, thanks for the reminder